examples/hosts/app-integration/external-data/src/view/appView.tsx

```diff
@@ -23,6 +23,17 @@ export interface IAppViewProps {
  */
 export const AppView: React.FC<IAppViewProps> = (props: IAppViewProps) => {
 	const { model } = props;
-
-	return <TaskListView taskList={model.taskList} />;
+	// The DebugView is just for demo purposes, to offer manual controls and inspectability for things that normally
```

Comment:
I had originally split the DebugView from the AppView so that the AppView could just be what a "normal" app would have, whereas the DebugView would hold all the demo-specific stuff that wouldn't show up in a real application.  I'd suggest either maintaining that separation or else doing a full merge of the two if the separation is no longer desired.

Reply:
We should maintain the separation -- there's a future task to split the Debug View completely out into a separate path of its own.